cypress/support/commands.js

```diff
@@ -48,3 +48,47 @@ function terminalLog(violations) {
 Cypress.Commands.add('checkA11yVvv', () => {
     cy.checkA11y(null, { rules: { region: { enabled: false } } }, terminalLog);
 })
+
+Cypress.Commands.add('addPersonToNotify', (p) => {
```

Comment:
I think it would be better to have this as a function in the test file rather than a command, unless you plan on using it elsewhere?

Reply:
Potentially has further uses in the future but not right now so I'll move to where its used 👍